src/utils/utils.js

```diff
@@ -0,0 +1,23 @@
+async function syncForEach(array, callback) {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array)
+  }
+}
+
+function asyncForEach(array, callback) {
```

Comment:
Using this seems to be the same as doing `array.map(callback)`. If that's the case, the abstraction is not worth it in my opinion.